package.json

```diff
@@ -25,6 +25,8 @@
     "@tiptap/extension-table-row": "^2.2.1",
     "@tiptap/extension-text-align": "^2.2.1",
     "@tiptap/extension-underline": "^2.2.1",
+    "@tiptap/extension-subscript": "^2.11.5",
+    "@tiptap/extension-superscript": "^2.11.5",
```

Comment:
When new npm dependencies are added, the package.lock file needs to be pushed to the repository.